concepticondata/references/references.bib

```diff
@@ -3623,3 +3623,11 @@ @article{Cowen2020
   Doi                                 = {10.1038/s41586-020-3037-7}
 }
 
+@article{Anonby2018,
+  Author                              = {Anonby, Erik and Taheri-Ardali, Mortaza and Hayes, Amos},
+  Title                               = {The Atlas of the Languages of Iran (ALI): A Research Overview},
+  Journal                             = {Iranian Studies},
+  Year                                = {2019},
+  Volume                              = {52},
+  Number                              = {1-2},
```

Comment:
I would also suggest adding the doi: 10.1080/00210862.2019.1573135.
and pages: 199 - 230
Taken from here: https://www.semanticscholar.org/paper/The-Atlas-of-the-Languages-of-Iran-(ALI)%3A-A-Anonby-Taheri-Ardali/d7769d7799e82afa87ba7d1c13c24e55edb4b5f0